tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -4126,6 +4126,79 @@ public function testHabtmSelectBox()
             '/div'
         ];
         $this->assertHtml($expected, $result);
+
+        $tags = [
+            1 => 'blue',
+            50 => 'green'
+        ];
+        $this->View->viewVars['tags'] = $tags;
```

Comment:
It might be better to use something like `people` here as that is a name that would break if passed through pluralize().